controllers/wavefront_controller_test.go

```diff
@@ -742,102 +703,36 @@ func TestReconcileProxy(t *testing.T) {
 			require.Equal(t, int32(2), *deployment.Spec.Replicas)
 		})
 
-		t.Run("defaults to one when no available proxy exists", func(t *testing.T) {
```

Comment:
This is no longer relevant because we don't spin up everything until the proxy is running.